### PR TITLE
Bookstore: Use Redux in React components.

### PR DESCRIPTION
**Pull request by andrianarivo**
In this PR we:

1. Wrapped `<App>` with the ` <Provider>` component
2. Passed my configured `store` into the `<Provider>`
3. Created an initial state
4. Displayed `books`
5. Dispatch actions using `useDispatch`
6. Added a <Button> component that adds a book to the books array (with attributes id, title and author)
7. Added a <Button> component that removes a book from the books array (by id)

### CODE REVIEW

File: src/components/BookItem.js [hunk 1]
```diff
@@ -25,4 +38,6 @@ export default function BookItem(props) {
 BookItem.propTypes = {
   id: PropTypes.number.isRequired,
```
Comment (OyePriscilla) [inline src/components/BookItem.js:39]: - [ ] When I tried to delete an item from the list, all the items were deleted and I got this 👇  error message from the console.  Kindly resolve it. ℹ️ 

Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @andrianarivo 👍, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Used Redux in React Components :mark
React component is wrapped with the <Provider>
Redux store has been passed into the <Provider>
Adding and removing books functionalities worked as expected 

## Required Changes ♻️
- [ ] Kindly work more on the README file, the live demo link and issues page are broken at the moment, **I was not redirected to the appropraite page respectively**. Implementing this will make your repo to be more informative and engaging. 👍 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookItem.js (hunk 1 above)
Comment (andrianarivo) [reply src/components/BookItem.js:39]: Thank you

**Review by t-yanick — APPROVED** [review]
# STATUS: MILESTONE APPROVED :1st_place_medal: 

Hi @andrianarivo , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Fixed remove book bug
:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file and PR

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**